src/common/kube-helpers.ts

```diff
@@ -16,16 +16,50 @@ function resolveTilde(filePath: string) {
   return filePath;
 }
 
+function loadFromFile(file: string) {
+  return loadFromString(fse.readFileSync(file, 'utf8'));
+}
+
+function loadFromString(config: string) {
```

Comment:
I would like to see a unit test covering this change

Reply:
Adding unit tests today and tomorrow.

Reply:
@Nokel81 - Can you please check the unit tests I have just added?